src/lib/commandSetup.ts

```diff
@@ -127,6 +127,7 @@ export class Connections {
     if (chain) {
       try {
         this._network = helpers.getChainInfo(chain);
+        /* c8 ignore next 3 */
```

Comment:
not sure why since i didn't touch this, but the coverage wasn't passing due to this...which doesn't look like it was an issue before

Reply:
weird...
There's a coverage report that you can look at in your browser.  If you checkout main and run `npm run coverage:report` you might be able to get more detail.